### DISCUSSION TITLE
Notification service Failure

### DISCUSSION THREAD

**Discussion by prabhunaveen** [project-sunbird/sunbird-devops#3540] [General]
@saiakhil46 @keshavprasadms 

As part for release-5.0.0, The Notification service is failing with below error.Kindly help.

`TASK [sunbird-deploy : helm install] *******************************************
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": "helm upgrade --install --atomic --timeout 10m notification /var/lib/jenkins/workspace/Deploy/pre-production/Kubernetes1/Notification/kubernetes/helm_charts/core/notification -n preprod",
    "delta": "0:10:06.524507",
    "end": "2022-09-20 07:25:48.603906",
    "rc": 1,
    "start": "2022-09-20 07:15:42.079399"
}

STDERR:

Error: UPGRADE FAILED: release notification failed, and has been rolled back due to atomic being set: timed out waiting for the condition
`

Thanks in advance !

cc: @anandvarada @srajasimman @gandham-santhosh

**Comment by beepdot** [comment]
Please share the pod logs when the deployment is ongoing.

**Reply by kalaipandiyank** [reply]
@keshavprasadms @santhosh-tg please find attached the pod log file
[notification_log.txt](https://github.com/project-sunbird/sunbird-devops/files/9609043/notification_log.txt)

**Reply by beepdot** [reply]
@reshmi-nair @amit-tarento Can you please check this error and let @kalaipandiyank know how to fix this?

**Reply by prabhunaveen** [reply]
@reshmi-nair @amit-tarento Kindly help us to fix the issue.

cc:  @anandvarada @srajasimman @keshavprasadms

**Reply by Hari-stackroute** [reply]
@prabhunaveen can you please schedule a call with me
cc: @reshmi-nair

**Reply by prabhunaveen** [reply]
@Hari-stackroute Sure, we will schedule call.

cc: @anandvarada @srajasimman @kalaipandiyank

**Reply by gandham-santhosh** [reply]
@prabhunaveen can you please mark this as answered if this is addressed? And also post the resolutions steps

**Reply by prabhunaveen** [reply]
@gandham-santhosh Now the pods are running with latest image.But still the deployment is failing.

@saiakhil46  Is already working on this issue, Kindly share the updates.

cc: @anandvarada @srajasimman
